orchestrator/cosmos/broadcast.go

```diff
@@ -77,11 +55,8 @@ type PeggyBroadcastClient interface {
 // sortableEvent exists with the only purpose to make a nicer sortable slice for Ethereum events.
 // It is only used in SendEthereumClaims
 type sortableEvent struct {
-	EventNonce         uint64
-	DepositEvent       *wrappers.PeggySendToCosmosEvent
-	WithdrawEvent      *wrappers.PeggyTransactionBatchExecutedEvent
-	ValsetUpdateEvent  *wrappers.PeggyValsetUpdatedEvent
-	ERC20DeployedEvent *wrappers.PeggyERC20DeployedEvent
+	EventNonce        uint64
+	ValsetUpdateEvent *wrappers.QuantumGravityBridgeValidatorSetUpdatedEvent
```

Comment:
The event `PeggyTransactionBatchExecutedEvent` was renamed in the contract to `MessageTupleRootEvent`. You could argue it's the only actual critical part of the contracts that needed to be modified. This PR is fine I think, but we need to replace the function that watched for `PeggyTransactionBatchExecutedEvent` with one that watches for `MessageTupleRootEvent` in the relayer (not sure why it's under the orchestrator).